lona/html/parsing.py

```diff
@@ -21,10 +28,88 @@
 ]
 
 
+def _find_node_classes(module):
+    try:
+        for attribute_name in dir(module):
+            node_class = getattr(module, attribute_name)
+
+            if not inspect.isclass(node_class):
+                continue
+
+            if not issubclass(node_class, Node):
+                continue
+
+            # input nodes
+            if node_class.TAG_NAME == 'input':
+                if 'type' not in node_class.ATTRIBUTES:
+                    logger.warning(
+                        'WARNING: input node %s has no type set',
+                        repr(node_class),
+                    )
+
+                    continue
+
+                INPUT_NODE_CLASSES[node_class.ATTRIBUTES['type']] = node_class
+
+            # nodes
+            else:
+                if node_class.TAG_NAME in NODE_CLASSES:
+                    logger.warning(
+                        'WARNING: Two Node classes with the same tag name were found: %s',  # NOQA
+                        node_class.TAG_NAME,
+                    )
+
+            NODE_CLASSES[node_class.TAG_NAME] = node_class
```

Comment:
nit: This line will be executed multiple times with same tag name `input` but different classes. Is it by intention?
```suggestion
                NODE_CLASSES[node_class.TAG_NAME] = node_class
```

Reply:
ACK! I fixed that